### ISSUE TITLE
Failed to create sound Effect: Crash while changing fonts

### ISSUE THREAD

**Issue by s-ol** [giann/settlers-issues#3]
after switching and saving fonts a couple of times this happened:

```
settlersrc not found, using default config
/home/s-ol/.local/share/love/settlers/settlersrc saved
/home/s-ol/.local/share/love/settlers/settlersrc saved
/home/s-ol/.local/share/love/settlers/settlersrc saved
/home/s-ol/.local/share/love/settlers/settlersrc saved
Error: src/soundboard.lua:1: Failed to create sound Effect.
stack traceback:
	lib/hump/gamestate.lua:103: in function <lib/hump/gamestate.lua:102>
	lib/hump/gamestate.lua:103: in function <lib/hump/gamestate.lua:102>
	lib/hump/gamestate.lua:103: in function <lib/hump/gamestate.lua:102>
	lib/hump/gamestate.lua:103: in function 'handler'
	[string "boot.lua"]:637: in function <[string "boot.lua"]:633>
	[C]: in function 'setEffect'
	src/soundboard.lua:1: in function 'init'
	lib/hump/class.lua:89: in function <lib/hump/class.lua:87>
	src/mainmenu.lua:1: in function 'push'
	src/mainmenu.lua:1: in function 'switch'
	src/defs.lua:1: in function 'initialize'
	src/ui/settingsmenu.lua:1: in function 'action'
	src/menu.lua:1: in function 'm'
	src/input.lua:1: in function 'matchCombo'
	src/input.lua:1: in function <src/input.lua:1>
	lib/hump/gamestate.lua:103: in function <lib/hump/gamestate.lua:102>
	lib/hump/gamestate.lua:103: in function <lib/hump/gamestate.lua:102>
	[string "boot.lua"]:503: in function <[string "boot.lua"]:493>
	[C]: in function 'xpcall'
```

this also happens (sometimes?) when starting a new game after changing fonts.

**Comment by QuantumQuacken** [comment]
I have noticed that after changing (and saving) font type for the third time the game crashes. Always. It keeps the new font you have selected right before crashing. 
Some system info:
```
System Information report written at: 29-Dec-2018 22:29:37
[System Summary]
OS Name	Microsoft Windows 10 Education	
Version	10.0.17134 Build 17134	
System Type	x64-based PC
Processor	Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz, 3301 Mhz, 4 Core(s), 4 Logical Processor(s)
BIOS Version/Date	American Megatrends Inc. P1.90, 7/27/2015
BIOS Mode	UEFI	

Installed Physical Memory (RAM)	16.0 GB	
Total Physical Memory	15.9 GB	
Available Physical Memory	11.3 GB	
Total Virtual Memory	18.3 GB	
Available Virtual Memory	12.6 GB	
Page File Space	2.38 GB		

NVIDIA GeForce GTX 950
Resolution	1920 x 1080 x 144 hertz
```
I posted it here since it is related with @s-ol initial problem.
If needed I'll open a different issue

**Comment by giann** [comment]
Fixed in future release
